src/st/st0/2F774.c

```diff
@@ -0,0 +1,254 @@
+#include "st0.h"
+
+void func_801AF774(Entity* self) {
+    RECT sp10;
+    DRAWENV sp18;
+    DRAWENV* var_a2;
+    DRAWENV* var_a3;
+    Primitive* prim;
+    s32 primIndex;
+    s32 i;
+    u32 var_t0;
+    u8 temp_t0;
+    u8* var_a2_3;
+    s16* temp_a3;
+    s32 xBase;
+    s32 yBase;
+    inline s32 add(a, b) { return a + b; }
+
+    switch (self->step) {
+    case 0:
+        InitializeEntity(D_801805E0);
+        self->hitboxState = 0;
+        self->animCurFrame = 0;
+        self->drawMode |= 0x30;
```

Comment:
```suggestion
        self->drawMode |= DRAW_TPAGE | 0x20;
```
unfortunately 0x20 and 0x40 are not bitfields. It's kinda weird. It is documented in `RenderPrimitives`